src/main/java/com/exactpro/th2/FixHandlerSettings.java

```diff
@@ -46,6 +49,9 @@ public class FixHandlerSettings implements IHandlerSettings {
     private String passwordEncryptAlgorithm = "RSA";
     private Boolean resetSeqNumFlag = false;
     private Boolean resetOnLogon = false;
+    private Boolean maintainSessionBasedOnNextExpectedSeqNumber = false;
+    private Boolean isSaveAdminMessages = false;
+    private String startOfADayTime;
```

Comment:
> private String startOfADayTime

Maybe rename to `sessionStartTime`?
And we should also have `sessionEndTime` setting too
Also I guess we can change its type to `LocalTime` - Jackson should be able to handle that (maybe certain annotations will be required)